Dockerfile.build

```diff
@@ -12,6 +12,7 @@ COPY yarn.lock /app/yarn.lock
 COPY package.json /app/package.json
 RUN yarn
 
+COPY intl.buildlangs.js /app/intl.buildlangs.js
```

Comment:
we'll need to add this to the Docker.storybooks too huh.

Reply:
nah, the build langs is only to generate the base english json -- so it's only needed in a dev environment